src/audio/base_fw.c

```diff
@@ -16,6 +16,10 @@
 #if defined(CONFIG_SOC_SERIES_INTEL_ADSP_ACE)
 #include <intel_adsp_hda.h>
 #endif
+#include <sof/debug/telemetry/telemetry.h>
```

Comment:
"sof/debug/telemetry/telemetry.h" is added couple commit later in "telemetry: Add systick_info part of telemetry", so this sure breaks build and bisect here.

Reply:
fixed